### ISSUE TITLE
sudo apt install libsuperlu5

### ISSUE THREAD

**Issue by gracexmatin** [uvahotspot/HotSpot#11]
![image](https://github.com/user-attachments/assets/690cd666-b434-4a5b-b255-fc062a78e410)
why I cannot successfully install libsuperlu5
will it make a big problem?

**Comment by satanu01** [comment]
No, I used Ubuntu 22 and installed using the following command:
`sudo apt install libblas-dev libsuperlu5 libsuperlu-dev`

Then set up the path in the Makefile:
```
++ SLU_HEADER   = /usr/include/superlu
++ SUPERLULIB   = -lsuperlu
++ BLASLIB      = -lblas
++ CFLAGS	= $(OFLAGS) $(EXTRAFLAGS) $(INCDIRFLAG) $(LIBDIRFLAG) -DVERBOSE=$(VERBOSE) -DMATHACCEL=$(ACCELNUM) -DDEBUG3D=$(DEBUG3D) -DSUPERLU=$(SUPERLU) -g -fcommon
```
Added the `-fcommon` flag; otherwise, it gives an error of repeated declareatio/defination when using the above gcc/g++-10.
